integration_tests/utils.go

```diff
@@ -290,3 +284,16 @@ func ExponentialBackOff(operation backoff.Operation, minutes time.Duration) erro
 
 	return backoff.Retry(operation, b)
 }
+
+func GetLocalHost(t *testing.T) string {
```

Comment:
The kubeconfig that is used to spin up the operator is the same that is used to create a Kubernetes account on spinnaker, so, we need to change the cluster URL.